otri/utils/__init__.py

```diff
@@ -1 +1,22 @@
-# Module used for imports
+import os
+import otri
+from setuptools import find_packages
+from pkgutil import iter_modules
+from pathlib import Path
+from typing import Set
+
+
+def get_otri_modules() -> Set[str]:
```

Comment:
What's the use of this?
Lol it sounds pretty passive-aggressive but I mean it, I'd like to know why you added it.